content/articles/firestore-data-modeling-and-query-customization-in-android/index.md

```diff
@@ -0,0 +1,302 @@
+### Introduction
+The Firestore database is a modern database which is object-based and uses JSON-format to store data. In this tutorial, we will learn how to prepare data, design the database, and perform simple and complex queries using the Android platform.
+
+### Prerequisites
+You'll need the following items to follow this tutorial:
+- You must have Android Studio installed.
+- Basic knowledge of Kotlin.
+- Be familiar with the Firebase Console. 
```

Comment:
I think Coroutines and ViewBinding knowledge is required too. I've seen you've implemented them in your code. Kindly include them and in this list.